timescaledb.control.in

```diff
@@ -1,5 +1,5 @@
 # timescaledb extension
-comment = 'Enables scalable inserts and complex queries for time-series data'
+comment = '(@LICENSE_EDITION@) Enables scalable inserts and complex queries for time-series data'
```

Comment:
I think i would prefer this being suffix instead of prefix

Reply:
It was added as prefix because this was a suggestion by product: https://github.com/timescale/timescaledb/issues/5436#issue-1623011404

Reply:
I also think a suffix would be easier to read as well (cc @iroussos).

Reply:
Adding the information as suffix then.